index.bs

```diff
@@ -2024,21 +2031,30 @@ parameter to the {{CredentialsContainer/create()}} or {{CredentialsContainer/get
     enum AuthenticatorTransport {
         "usb",
         "nfc",
-        "ble"
+        "ble",
+        "internal"
     };
 </pre>
 
 <div dfn-type="enum-value" dfn-for="AuthenticatorTransport">
-    Authenticators may communicate with clients using a variety of transports. This enumeration defines a hint as to how clients
-    might communicate with a particular authenticator in order to obtain an assertion for a specific credential. Note that these
-    hints represent the [=[RP]=]'s best belief as to how an authenticator may be reached. A [=[RP]=] may obtain a list of
-    transports hints from some [=attestation statement formats=] or via some out-of-band mechanism; it is outside the scope of
-    this specification to define that mechanism.
+    Authenticators may communicate with clients using a variety of transports. This enumeration defines a hint as to how
+    clients might communicate with a particular authenticator. It is used in two contexts: from client to RP when
+    [creating a new credential](#createCredential), and from RP to client when when [requesting an
+    assertion](#getAssertion).
+
+    In the first context, the RP might store this information in order to present appropriate instructions to the user
+    in the future. For example, instructing the user to press an NFC authenticator to the reader. (A [=[RP]=] may also
+    obtain a list of transports hints from some [=attestation statement formats=] or via some other out-of-band
+    mechanism; it is outside the scope of this specification to define such mechanisms.)
+
+    In the second context, this is a hint representing the [=[RP]=]'s best belief as to how an authenticator may be
+    reached.
```

Comment:
Suggest a modest re-write of the abovce such that terms are aligned with other portions of the spec:
[=Authenticators=] may implement various [[#transport|transports]] for communicating with [=clients=]. This enumeration defines hints as to how clients might communicate with a particular authenticator. It is used during both [[#createCredential|credential creation]] and [[#getAssertion|assertion generation]].

During [[#createCredential|credential creation]], the RP can store the hint in order to present appropriate instructions to the user in the future. For example, instructing the user to press an NFC authenticator to the reader. 

Note: A [=[RP]=] may also obtain a list of transports hints from some [=attestation statement formats=] or via some other out-of-band mechanism (e.g., [[FIDOMetadataService]]; it is outside the scope of this specification to define such mechanisms.

Later, during [[#getAssertion|assertion generation]], the [=[RP]=] can supply the hint to the [=[client]=] to facilitate contacting the [=authenticator=]. The [=[RP]=] can also use the hint to appropriately tailor the user experience.

Reply:
s/During [[#createCredential|credential creation]]/After completing [[#createCredential|credential creation]]/
?

Reply:
Done.